Dockerfile

```
@@ -0,0 +1,87 @@
+FROM rocker/tidyverse:4.2
+MAINTAINER rokita@chop.edu
```

```suggestion
MAINTAINER rokita@chop.edu
```

Since this is going to be stored in your CAVATICA registry, you could put yourself here. Though it doesn't matter per se for this. Please add me as an admin for your docker image, so that I can make changes as well and/or add others to contribute. Thanks!